flang/lib/Frontend/FrontendActions.cpp

```diff
@@ -851,11 +851,10 @@ getOutputStream(CompilerInstance &ci, llvm::StringRef inFile,
 /// \param [in] act Backend act to run (assembly vs machine-code generation)
 /// \param [in] llvmModule LLVM module to lower to assembly/machine-code
 /// \param [out] os Output stream to emit the generated code to
```

Comment:
Nit: Add the new option here. May be good to have the stream as the last option.